2018-12-13 15:57:13,026 INFO 186431[guizhou_start.py:18 start_cmd]: --------------------------------------------------------------------------------------------
2018-12-13 15:57:13,026 INFO 186431[guizhou_start.py:19 start_cmd]: now start the host : 10.11.5.80
2018-12-13 15:57:13,032 INFO 186431[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-12-13 15:57:13,119 INFO 186431[transport.py:1746 _log]: Authentication (publickey) failed.
2018-12-13 15:57:13,143 INFO 186431[transport.py:1746 _log]: Authentication (password) successful!
2018-12-13 15:57:13,893 INFO 186431[guizhou_start.py:25 start_cmd]: cmd is : sudo /data/CI/kpms_30_13410/kpms.sh start
2018-12-13 15:57:13,893 INFO 186431[guizhou_start.py:26 start_cmd]: error list start is :[]
2018-12-13 15:57:13,894 INFO 186431[guizhou_start.py:27 start_cmd]: stdout list start is :[u'/data/CI/kpms_30_13410\n', u'kpms.war\n', u'kpms.war port: 13410 is exist. \n', u'kpms.war start failed!\n']
2018-12-13 15:57:13,894 INFO 186431[guizhou_start.py:28 start_cmd]: process is starting,wait a momunt
[Finish] Success to deploy [818]